components/raftstore/src/store/fsm/apply.rs

```diff
@@ -1087,7 +1087,7 @@ where
         self.metrics.written_keys += apply_ctx.delta_keys();
     }
 
-    fn write_apply_state(&self, wb: &mut EK::WriteBatch) {
+    fn force_write_apply_state(&self, wb: &mut EK::WriteBatch) {
```

Comment:
I think the original name should be kept.

Reply:
fixed